data_structures/src/vrf.rs

```diff
@@ -140,6 +154,10 @@ impl BlockEligibilityClaim {
                 Hash::SHA256(sha256)
             })
     }
+    /// Create a VRF proof for a given message. Needed to generate the input for following eligibilities.
```

Comment:
Looks to me like this comment could use an empty line on top.
Also I can't understand what "for following eligibilities" means.